lib/sample_rate_converter/sample_rate_converter_filter.c

```diff
@@ -71,6 +164,23 @@ int sample_rate_converter_filter_get(enum sample_rate_converter_filter filter_ty
 		}
 		break;
 #endif /* CONFIG_SAMPLE_RATE_CONVERTER_FILTER_TEST */
+#if CONFIG_SAMPLE_RATE_CONVERTER_FILTER_SMALL
+	case SAMPLE_RATE_FILTER_SMALL:
+		if (conversion_ratio == -2) {
+			*filter_ptr = filter_48khz_to_24khz_16bit_small;
+			*filter_size = ARRAY_SIZE(filter_48khz_to_24khz_16bit_small);
+		} else if (conversion_ratio == 2) {
+			*filter_ptr = filter_24khz_to_48khz_16bit_small;
+			*filter_size = ARRAY_SIZE(filter_24khz_to_48khz_16bit_small);
+		} else if (conversion_ratio == -3) {
+			*filter_ptr = filter_48khz_to_16khz_16bit_small;
+			*filter_size = ARRAY_SIZE(filter_48khz_to_16khz_16bit_small);
+		} else if (conversion_ratio == 3) {
+			*filter_ptr = filter_16khz_to_48khz_16bit_small;
+			*filter_size = ARRAY_SIZE(filter_16khz_to_48khz_16bit_small);
+		}
+		break;
```

Comment:
Should be a switch case, and you could create an enum for the values -3, -2, 2 ,and 3